cmd/fleetctl/logout_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLogout(t *testing.T) {
+	_, ds := runServerWithMockedDS(t)
+
+	ds.SessionByIDFunc = func(ctx context.Context, id uint) (*fleet.Session, error) {
+		return &fleet.Session{
+			ID:         333,
+			AccessedAt: time.Now(),
+			UserID:     123,
+			Key:        "12344321",
+		}, nil
+	}
+	ds.DestroySessionFunc = func(ctx context.Context, session *fleet.Session) error {
+		return nil
+	}
+
+	assert.Equal(t, "", runAppForTest(t, []string{"logout"}))
```

Comment:
Should probably check that the mocks got called, to make sure it runs as expected.